tracer/build/_build/Build.Steps.cs

```diff
@@ -318,6 +322,47 @@ partial class Build
 
             }
         });
+    
+    Target CopyLibDdwafForAppSecUnitTests => _ =>
+    {
+        void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath absolutePath, string wafFileName, string extension)
+        {
+            var source = LibDdwafDirectory / "runtimes" / architecture / "native" / $"{wafFileName}.{extension}";
+            var nativeDir = DDTracerHomeDirectory / architecture / $"Datadog.Trace.ClrProfiler.Native.{extension}"; 
+            foreach (var fmk in frameworks)
+            {
+                var dest = absolutePath / "bin" / BuildConfiguration / fmk / architecture;
+                CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
+                if (!IsWin)
+                {
+                    CopyFileToDirectory(nativeDir,  absolutePath / "bin" / BuildConfiguration / fmk, FileExistsPolicy.Overwrite);
+                }
+            }
+        }
+
+        return _.Unlisted()
+                .After(Clean)
+                .After(DownloadLibDdwaf)
+                .OnlyWhenStatic(() => !IsArm64)// not supported yet
+                .Executes(() =>
+                 {
+                     var project = Solution.GetProject(Projects.AppSecUnitTests);
+                     var directory = project.Directory;
+                     var targetFrameworks = project.GetTargetFrameworks();
+                     if (IsWin)
+                     {
+                         foreach (var architecture in WafWindowsArchitectureFolders)
+                         {
+                             CopyWaf(architecture, targetFrameworks, directory, "ddwaf", "dll");
+                         }
+                     }
+                     else
+                     {
+                         var (architecture, ext) = GetUnixArchitectureAndExtension();
+                         CopyWaf(architecture, targetFrameworks, directory, "libddwaf", ext);
+                     }
+                 });
+    };
```

Comment:
I think there's been some weird refactoring here (suggested fix below), but it seems kinda weird this is required for unit tests? Does that mean you can't run Unit tests from the IDE now?

```suggestion
    Target CopyLibDdwafForAppSecUnitTests => _ =>
        .Unlisted()
        .After(Clean)
        .After(DownloadLibDdwaf)
        .OnlyWhenStatic(() => !IsArm64)// not supported yet
        .Executes(() =>
        {
            var project = Solution.GetProject(Projects.AppSecUnitTests);
            var directory = project.Directory;
            var targetFrameworks = project.GetTargetFrameworks();
            if (IsWin)
            {
                foreach (var architecture in WafWindowsArchitectureFolders)
                {
                    CopyWaf(architecture, targetFrameworks, directory, "ddwaf", "dll");
                }
            }
            else
            {
                var (architecture, ext) = GetUnixArchitectureAndExtension();
                CopyWaf(architecture, targetFrameworks, directory, "libddwaf", ext);
            }

            static void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath absolutePath, string wafFileName, string extension)
            {
                var source = LibDdwafDirectory / "runtimes" / architecture / "native" / $"{wafFileName}.{extension}";
                var nativeDir = DDTracerHomeDirectory / architecture / $"Datadog.Trace.ClrProfiler.Native.{extension}"; 
                foreach (var fmk in frameworks)
                {
                    var dest = absolutePath / "bin" / BuildConfiguration / fmk / architecture;
                    CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
                    if (!IsWin)
                    {
                        CopyFileToDirectory(nativeDir,  absolutePath / "bin" / BuildConfiguration / fmk, FileExistsPolicy.Overwrite);
                    }
                }
            }
        });
```

Reply:
You'll need to run this at least once for some of the appsec security tests to run. After this, you should be able to run them from the gui, since ddwaf.dll will change rarely as it an external project. As appsec has it's own unit test projects this should mean that no tracer unit tests will be impacted.